great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -377,8 +362,105 @@ class MetricDomainTypes.
                 data = data.filter(parsed_condition)
             else:
                 raise GreatExpectationsError(
-                    f"unrecognized condition_parser {str(condition_parser)}for Spark execution engine"
+                    f"unrecognized condition_parser {str(condition_parser)} for Spark execution engine"
+                )
+
+        if "column" in domain_kwargs:
+            return data
+
+        if (
+            "column_A" in domain_kwargs
+            and "column_B" in domain_kwargs
+            and "ignore_row_if" in domain_kwargs
+        ):
+            # noinspection PyPep8Naming
+            column_A_name = domain_kwargs["column_A"]
+            # noinspection PyPep8Naming
+            column_B_name = domain_kwargs["column_B"]
+
+            ignore_row_if = domain_kwargs["ignore_row_if"]
+            if ignore_row_if == "both_values_are_missing":
+                ignore_condition = (
+                    F.col(column_A_name).isNull() & F.col(column_B_name).isNull()
+                )
+                data = data.filter(~ignore_condition)
+            elif ignore_row_if == "either_value_is_missing":
+                ignore_condition = (
+                    F.col(column_A_name).isNull() | F.col(column_B_name).isNull()
+                )
+                data = data.filter(~ignore_condition)
+            else:
+                if ignore_row_if != "never":
+                    raise ValueError(
+                        f'Unrecognized value of ignore_row_if ("{ignore_row_if}").'
+                    )
+
+            return data
+
+        if "column_list" in domain_kwargs and "ignore_row_if" in domain_kwargs:
+            column_list = domain_kwargs["column_list"]
+            ignore_row_if = domain_kwargs["ignore_row_if"]
+            if ignore_row_if == "all_values_are_missing":
+                ignore_condition = reduce(
+                    lambda column_a, column_b: column_a.isNull() & column_b.isNull(),
+                    map(F.col, column_list),
+                )
```

Comment:
fancy! is there a simpler-to-read version of this? I can't think of one, but wish there is just an "or" like in sql...